tests/api/api/test_runs.py

```diff
@@ -224,6 +224,54 @@ def test_list_runs_times_filters(db: Session, client: TestClient) -> None:
     )
 
 
+def test_list_completed_runs(db: Session, client: TestClient) -> None:
+    runs_endpoint = f"/api/projects/{config.default_project}/completed_runs"
+    base_time = datetime.now(timezone.utc)
+    total_runs = 10
+
+    runs = []
+    counter = 0
+    while counter < total_runs:
+        name = f"name_{counter}"
+        uid = f"id_{counter}"
+        run = Run(
+            name=name,
+            uid=uid,
+            project=config.default_project,
+            iteration=0,
+            start_time=base_time - timedelta(hours=counter * 2),
+        )
+        run.struct = {
+            "metadata": {"name": name, "uid": uid},
+        }
+        runs.append(run)
+        counter += 1
+
+    get_db()._upsert(db, runs, ignore=True)
+
+    # Assert that the run started at base_time is available instantly
+    assert_time_range_request(
+        client,
+        [runs[0].uid],
+        runs_endpoint=runs_endpoint,
+        since_datetime=datetime.isoformat(base_time),
+    )
+
+    # Assert that all other runs can be filtered successfully
+    counter = 1
+    uids = [r.uid for r in runs]
+    while counter <= total_runs:
```

Comment:
Same here, `for counter in range(total_runs):` and then use the `counter + 1`

Reply:
Indeed! I replaced this one as well.